python/paddle/fluid/layers/control_flow.py

```diff
@@ -952,6 +949,11 @@ def less_than(x, y, force_cpu=None, cond=None):
 
     Returns:
         ${out_comment}.
+
+    Examples:
+        .. code-block:: python
+
+          out = fluid.layers.less_than(x=label, y=limit)
```

Comment:
There should have a test case, for example:
```
label = ..
limit = .. 
out_var = fluid.layers.less_than(x=label, y=limit)
...
result =exe.run(fetch_list=[out_var], feed_list=[...])

```

Reply:
Repair completed, thanks for your correction.